.github/workflows/ci.yaml

```diff
@@ -36,5 +36,6 @@ jobs:
           flake8
       - name: Test
         run: |
+          DJANGO_SETTINGS_MODULE=flow_results.testsettings python manage.py makemigrations --dry-run | grep 'No changes detected' || (echo 'There are changes which require migrations.' && exit 1)
```

Comment:
There's now a `--check` flag for `makemigration` that we can use instead: https://docs.djangoproject.com/en/4.2/ref/django-admin/#cmdoption-makemigrations-check